install/ansible/install.sh

```diff
@@ -131,6 +131,9 @@ cp /var/contiv/key.pem /ansible/roles/auth_proxy/files/
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```

Comment:
We shouldn't exclude it. veplugin in legacy swarm can be installed. We will not officially support it